qiskit/ignis/verification/quantum_volume/fitters.py

```diff
@@ -308,6 +316,51 @@ def plot_qv_data(self, ax=None, show_plt=True):
         if show_plt:
             plt.show()
 
+    def plot_qv_trial(self, depth, trial_index, figsize=(7, 5)):
+        """Plot individual trial.
+
+        Args:
+            depth(int): circuit depth
+            trial_index(int): trial index
+            figsize (tuple): Figure size in inches.
+
+        Returns:
+            matplotlib.Figure:
+                A figure for histogram of ideal and experiment probabilities.
+        """
+        circ_name = f"qv_depth_{depth}_trial_{trial_index}"
+        ideal_data = self._all_output_prob_ideal[circ_name]
+        exp_data = self._circ_counts[circ_name]
+
+        fig = plt.figure(figsize=figsize)
+        ax1 = plt.gca()
+
+        # plot experimental histogram
+        plot_histogram(exp_data, legend=['Exp'], ax=ax1)
+
+        # plot idea histogram overlap with experimental values
+        plot_histogram(ideal_data, legend=['Ideal'], bar_labels=False, ax=ax1, color='r')
+
+        # get ideal histograms and change to unfilled
+        bars = [r for r in ax1.get_children() if isinstance(r, Rectangle)]
+        for i in range(int(len(bars)/2), len(bars)-1):
+            bars[i].fill = False
+            # set non-black edge color to increase bar labels legibility
+            bars[i].set_edgecolor('saddlebrown')
+
+        # show experimental heavy output probability to the legend
+        ax1.plot([], [], ' ', label=f'HOP~{self._heavy_output_prob_exp[circ_name]:.3f}')
+
+        # plot median probability
+        median_prob = self._median_probabilities([self._all_output_prob_ideal[circ_name]])
+        ax1.axhline(median_prob, color='r', linestyle='dashed', linewidth=1, label='Median')
+        ax1.legend()
+        ax1.set_title(f'Quantum Volume {2**depth}, Trial #{trial_index}', fontsize=14)
+
+        plt.close(fig)  # close additional figure
```

Comment:
This needs to be updated in the same way as in #495 and #493

Reply:
Sorry I missed that. I have made the changes.